NICE.Identity.Management/ClientApp/src/data/endpoints.ts

```diff
@@ -2,6 +2,8 @@ const API_BASE_URL = process.env.REACT_APP_API_BASE_URL
 	? process.env.REACT_APP_API_BASE_URL
 	: "#{REACT_APP_API_BASE_URL}";
 
+const APP_BASE_URL = API_BASE_URL.replace("/api", '');
+
```

Comment:
What does this do exactly?

Reply:
It accesses the account controller to get the status endpoint which essentially returns JSON data containing the username.

Reply:
I've added it as a comment for future reference.